docs/content/contributing/submitting-issues.md

```diff
@@ -33,30 +33,31 @@ Examples:
 
 ## Feature Request
 
-Traefik is an open source project and aims to be the best edge router possible. 
+Traefik is an open source project and aims to be the best edge router possible.
 
 Remember when asking for new features that these must be useful to the majority (and not only useful in edge case scenarios, or hack-like setups).
 Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE/feature-request.yml) as much as possible.
 
 Do your best to explain what you're looking for, and why it would improve Traefik for everyone.
 Be detailed and share the use-case(s) to allow us to see the value of your feature request as quickly as possible.
-Features with a lot of positive interaction (claps, +1s, conversation about how this would impact them) indicate higher community interest and help us to prioritize.  
 
-If you are interested in creating a PR for your feature request, let us know in the the issue so we can work with you.  
+Features with a lot of positive interaction (claps, +1s, conversation about how this would impact them) indicate higher community interest and help us to prioritize.
+
+If you are interested in creating a PR for your feature request, let us know in the issue, so we can work with you.
 It can take a lot of work to make sure a PR can integrate with our existing code and planning with the team ahead of time can make sure that your PR can be accepted and merged quickly.
 
 ## Issues or Possible Bug Reports
 
 Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml) as much as possible.
 
 Explain the conditions in which you encountered the issue; what is your context?
-Share any logs you may have and make sure to share the steps it takes to reproduce your issue or bug. 
+Share any logs you may have, and make sure to share the steps it takes to reproduce your issue or bug.
 
 Remain as clear and concise as possible.
 
-Take time to polish the format of your message so we'll enjoy reading it and working on it. 
+Take time to polish the format of your message, so we'll enjoy reading it and working on it.
 Help your readers focus on what matters and help them understand the structure of your message (see the [GitHub Markdown Syntax](https://docs.github.com/en/get-started/writing-on-github)).
 
 ## International English
 
-Every maintainer / Traefik user is not a native English speaker, so if you feel sometimes that some messages sound rude, remember that it probably is a language barrier problem from someone willing to help you.
+Every maintainer / Traefik user is not a native English speaker, if you sometimes feel that some messages sound rude, remember that it probably is a language barrier problem from someone willing to help you.
```

Comment:
I disagree with this change. "so" here provides a logical link between the statements. Removing it makes the whole sentence a bit harder to understand.

Reply:
Interesting, for me, it is the way around.

In general, I prefer to be really carefully with the usage of “so” and “so if”, depending on the context and the background of the reader is can be potential harmful.

Ok, I will add it back for now.
That being said, 😄, I will rewrite that whole part in the future 😄 .

Reply:
cool, thanks.